### ISSUE TITLE
Mention whatsnew character limit in README

### ISSUE THREAD

**Issue by hzsweers** [Triple-T/gradle-play-publisher#41]
Just a suggestion, as the API error response isn't very descriptive. The "what's new" text can be a max of **80 characters** (as it's replacing the "Promo Text" field in the store listing").

The response if you exceed this looks like this:

```
> com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
  {
    "code" : 403,
    "errors" : [ {
      "domain" : "androidpublisher",
      "message" : "The recent changes text for APK version 2565 is too long for language .",
      "reason" : "apkRecentChangesTooLong"
    } ],
    "message" : "The recent changes text for APK version 2565 is too long for language ."
  }
```

**Comment by hzsweers** [comment]
Alternatively, you could also catch this in the build itself and throw your own (more descriptive) exception and avoid making the API call.

**Comment by hzsweers** [comment]
Actually nevermind, the 80 chars isn't correct and I see you handle this in the code already. That said, the code [here](https://github.com/Triple-T/gradle-play-publisher/blob/master/src%2Fmain%2Fgroovy%2Fde%2Ftriplet%2Fgradle%2Fplay%2FPlayPublishApkTask.groovy) doesn't seem to actually trim it correctly

**Comment by bhurling** [comment]
Fixed in 1.0.2

**Comment by hzsweers** [comment]
Thanks for the quick update :)
